articles/introduction-to-solid-principle/index.md

```diff
@@ -0,0 +1,255 @@
+# Introduction to Solid Principle
+Writing clean codes is one of the core precepts of software development. There are various software design approaches to ensure an understandable, flexible, and maintainable code base.
+
+This article explains one of the object-oriented programming design principles that assist you as a software developer towards writing clean codes, without narrowing its application to any programming language, but from a universal point of view towards understanding the concept of the SOLID principle.
+
+# What is Solid Principle?
+The SOLID principle is a design principle and pattern intended to make software designs more understandable, flexible, and maintainable. In the software development process, when creating an object ‒ a solid object or an abstract object, this object's design is essential, notably in deciding the accessibility scope.
+
+The object case scenario might confuse newbies in software development. Here's a better analogy; imagine if cups had their handles at the top rather than at the side. How accessible or comfortable would it be to use this cup (object)? But if the handle remains at the side, they remain readily accessible and comfortable to use.
+
+In software development, the decision on how accessible and flexible an object is should occur during this object's design phase. This decision-making process pre-software development ensures usability, simplicity, ease of implementation, and accessibility towards making software better than others.
```

Comment:
Some clarity on the second sentence is required. Rewrite the sentence into 2 smaller sentences.